custom/icds/location_reassignment/tests/test_download.py

```diff
@@ -80,8 +80,10 @@ def test_dump(self, get_location_mock, get_household_case_ids_mock, case_accesso
             call('1'), call('2')
         ])
         child_cases_mock.assert_has_calls([
-            call(self.domain, '1', location.location_id, [PERSON_CASE_TYPE]),
-            call(self.domain, '2', location.location_id, [PERSON_CASE_TYPE])
+            call(self.domain, '100', location.location_id, [PERSON_CASE_TYPE]),
+            call(self.domain, '100', location.location_id, [PERSON_CASE_TYPE]),
+            call(self.domain, '100', location.location_id, [PERSON_CASE_TYPE]),
+            call(self.domain, '100', location.location_id, [PERSON_CASE_TYPE])
```

Comment:
Since we are [case.case_id](https://github.com/dimagi/commcare-hq/pull/27734/commits/f9d77620753626c53b4417a65c650a049e0b511d#diff-af1942757ada9f313046092629aac433R279) so the change for '1' (mocked value) to '100`(case id put on the case mocked to return for any case access)

Also, there are 4 calls instead of 2 for real. At some point the number of transitions were increased for test, but the corresponding calls were not increased. `assert_has_calls` does not fail if the expected calls are less than original calls but only if they don't match or are more.

Reply:
More calls updated later, https://github.com/dimagi/commcare-hq/pull/27734/commits/55ab0be3132abe76bd06ebf4dff952cd5d9cb01b